zipline/assets/__init__.py

```diff
@@ -20,6 +20,15 @@
     make_asset_array,
     CACHE_FILE_TEMPLATE
 )
+from .asset_writer import (
+    NullAssetDBWriterLegacy,
+    AssetDBWriterLegacyFromList,
+    AssetDBWriterLegacyFromDictionary,
+    AssetDBWriterLegacyFromDataFrame,
+    AssetDBWriterLegacyFromReadable,
+    FUTURE_TABLE_FIELDS,
+    EQUITY_TABLE_FIELDS
```

Comment:
Yup, they're now defined in the writer (asset_writer.py) but also used in the reader (assets.py), hence the reader needs to import them.